Source/Core/Event.js

```diff
@@ -120,6 +120,10 @@ define([
         return false;
     };
 
+    function compareNumber(a,b) {
```

Comment:
Since this is so short, it can probably be inline.


```
toRemove.sort(function (a, b) {
    ...
});
```

Reply:
Wouldn't that add an unnecessary callback allocation every time an event is raised?

Reply:
That's true, fair enough!